### PR TITLE
Add labels documentation

### PR DESCRIPTION
**Pull request by BenVillalobos**
### Context
Added a doc with a table explaining some of the more frequently used labels, and the underlying processes that apply to them.

### Notes
Once this is merged I'll update the bot's "close a stale issue" message with a link to the doc.

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -20,6 +20,8 @@
 
     <ProduceReferenceAssembly Condition="'$(IsTestProject)' != 'true'">true</ProduceReferenceAssembly>
 
+    <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
```
Comment (rainersigwald) [inline src/Directory.Build.props:23]: You shouldn't need this. Are you on RC2 locally?

**Review by rainersigwald — COMMENTED** [review]
LGTM overall. Worth linking from root `README.md`?

File: src/Directory.Build.props (hunk 1 above)
Comment (rainersigwald) [reply src/Directory.Build.props:23]: Just realized when you responded that you only need to be on RC1 for this, and dogfood VS should give it to you automagically.

File: documentation/wiki/Labels.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# MSBuild Labels
+Here's a brief explanation on the labels most often used by the MSBuild team excluding obvious ones such as `bug`.
```
Comment (rainersigwald) [inline documentation/wiki/Labels.md:2]:
```suggestion
Here's a brief explanation on the labels most often used by the MSBuild team excluding hopefully-self-evident ones such as `bug`.
```

"Obvious" is a dangerous word for public writing: if it wasn't obvious to the reader they feel bad.

**Review by rainersigwald — APPROVED** [review]

File: src/Directory.Build.props (hunk 1 above)
Comment (AR-May) [reply src/Directory.Build.props:23]: @BenVillalobos, have you forgotten to remove this line from PR? Please do it before I merge.

Comment (BenVillalobos) [reply src/Directory.Build.props]: Whoops, removed it. Thanks for double checking!

**Review by AR-May — APPROVED** [review]

**Pull request merged**